menelaos-front/README.md

```diff
@@ -0,0 +1,38 @@
+# MenelaosFront
```

Comment:
Unnecessary file, should be removed. The content probably could be moved to ../README.md

Reply:
Done